### PR TITLE
Migrate `inspector_test` to null-safety

### PR DESCRIPTION
**Pull request by elliette**

### CODE REVIEW

**Comment by elliette** [comment]
Depends on changes in https://github.com/dart-lang/webdev/pull/1709

File: dwds/test/inspector_test.dart [hunk 1]
```diff
@@ -160,7 +158,7 @@ void main() {
 
     test('invoke instance private', () async {
       final remote = await inspector.invoke(
-          instance.objectId, 'privateMethod', [dartIdFor('some string')]);
+          instance.objectId!, 'privateMethod', [dartIdFor('some string')]);
```
Comment (annagrin) [inline dwds/test/inspector_test.dart:161]: Looks like instance.objectId! is used in many places, would be great to have a non-nullable objectId variable instead.

**Review by annagrin — APPROVED** [review]
LGTM with comments

File: dwds/test/inspector_test.dart (hunk 1 above)
Comment (elliette) [reply dwds/test/inspector_test.dart:161]: Done

**Pull request merged**